sci-physics/vgm/vgm-4.8.ebuild

```diff
@@ -8,24 +8,27 @@ inherit cmake
 if [[ ${PV} == *9999* ]]; then
 	inherit git-r3
 	EGIT_REPO_URI="https://github.com/vmc-project/${PN}.git"
+	KEYWORDS=""
 else
-	SRC_URI="http://ivana.home.cern.ch/ivana/${PN}.${PV}.tar.gz"
-	KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
-	S="${WORKDIR}/${PN}.${PV}"
+	MY_PV=$(ver_rs 1- -)
+	SRC_URI="https://github.com/vmc-project/${PN}/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/${PN}-${MY_PV}"
 fi
 
 DESCRIPTION="Virtual Geometry Model for High Energy Physics Experiments"
 HOMEPAGE="http://ivana.home.cern.ch/ivana/VGM.html"
 
 LICENSE="GPL-2"
 SLOT="0"
-IUSE="doc examples +geant4 +root test"
+IUSE="+c++11 c++14 c++17 doc examples +geant4 +root test"
+
+REQUIRED_USE="^^ ( c++11 c++14 c++17 )"
 
-# sci-physics/root[c++11] required to match sci-physics/geant
 RDEPEND="
 	sci-physics/clhep:=
-	root? ( sci-physics/root:=[c++11] )
-	geant4? ( >=sci-physics/geant-4.10.03 )"
+	geant4? ( >=sci-physics/geant-4.10.06[c++11?,c++14?,c++17?] )
```

Comment:
4.10.6, sorry. I changed a bit how the versions are named. The only place the that padding zeros are used by Geant4 is in the tarball name, so I think that dropping the zeros was better. If you think it's confusing I can go back to the old style. At least [repology](https://repology.org/projects/?category=sci-physics&maintainer=amadio%40gentoo.org) seems confused about it..

Reply:
Even switching to sci-physics/geant4 makes sense in that case... https://repology.org/project/geant4/versions

Reply:
Indeed! I changed it now, thanks for pointing this out (for the dependency, it seems portage interprets 06 as 6 anyways, but better be consistent here). 
I also found https://repology.org/project/geant4-vmc/versions on related matters, but this seems to be something outdated for one distro only, so I guess for `sci-physics/geant-vmc` we should stay without the trailing `4`. 
For Geant4, the versioning is also inconsistent on repology — MacPorts seems to package Geant4 4.10.6 (that numbering looks completely broken to me).